libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -866,6 +866,40 @@ button.gd-button-primary {
     }
 }
 
+.gd-invite-button {
```

Comment:
Could you please refer to the style of Help Button or Account Button on the Header to make consistency?

Reply:
I updated it thanks